<launch>
	<arg name="robot" default="pr2"/>
	
	<node pkg="grasp_template_planning" type="main_dataset_from_point_clouds" name="main_dataset_from_point_clouds" output="screen">
		<param name="dir_path_base" type="str" value="$(find grasp_template_planning)"/>
		<rosparam file="$(find grasp_template_planning)/config/template_config_$(arg robot).yaml" command="load"/>
	</node>
</launch>
